src/Servers/IIS/AspNetCoreModuleV2/InProcessRequestHandler/managedexports.cpp

```diff
@@ -581,10 +582,11 @@ http_response_set_trailer(
     _In_ IN_PROCESS_HANDLER* pInProcessHandler,
     _In_ PCSTR pszHeaderName,
     _In_ PCSTR pszHeaderValue,
-    _In_ USHORT usHeaderValueLength)
+    _In_ USHORT usHeaderValueLength,
```

Comment:
Why does this API need to know the value length but not the name length?

Reply:
So it looks like we're calling `strlen(pszHeaderName)` internally. I don't see any reason why we can't do the same for `pszHeaderValue` and eliminate `usHeaderValueLength`. I think maybe Levi didn't really have to chance to clean it up, and I didn't notice it until you pointed it out. I can remove the length parameter if we think it's important.

Reply:
Yes please. What do the other header APIs do?